src/_includes/layouts/listing-blog-post-categories.njk

```diff
@@ -38,15 +38,69 @@
 }
 ---
 
-<h1>{{ title }} "{{ tag }}"</h1>
-
 {% set blogPostLocaleTag = locale + '-blog-post' %}
+{% set articleLocaleTag = locale + '-article' %}
+{% set blogPostFilters = collections[blogPostLocaleTag] | collectionTags([blogPostLocaleTag, articleLocaleTag]) %}
 {% set blogPostCategories = collections[tag] | collectionIncludesTag(blogPostLocaleTag) | reverse %}
 
-<ul class="ul grid grid--cols-4">
-  {% for item in blogPostCategories %}
-    <li>
-      {% include "components/blog-post-card.njk" %}
-    </li>
-  {% endfor %}
-</ul>
\ No newline at end of file
+<section class="section">
```

Comment:
Thought: Feels like the bulk of the templated part of this layout is the same whether a blog-category/blog-list page. Opportunity to make a generic blog list template that both could inherit from? (With just a smidge of conditional logic to omit the "Articles filtered by..." title). 

Maybe a premature optimisation, but feels some could be DRYed out here

Reply:
@adamduncan The big difference between those layouts is the pagination. 

`listing-blog-posts` uses pagination in the traditional sense that it creates multiple pages of posts where as `listing-blog-categories` uses it to automate the creation of a page for each individual category based on a tag.

Maybe we can look at this as a future optimisation but I think for now we should leave as is.